docs-cn/14-reference/03-connector/node.mdx

```diff
@@ -201,3 +201,53 @@ promise2.then(function (result) {
 [node-example-raw.js](https://github.com/taosdata/tests/tree/master/examples/nodejs/node-example-raw.js)同样是一个使用 NodeJS 连接器建表，插入天气数据并查询插入的数据的代码示例，但和上面不同的是，该示例只使用`cursor`。
 
 [所有示例代码](https://github.com/taosdata/TDengine/tree/develop/src/connector/nodejs/examples)
+
+## 使用RESTful 连接器
```

Comment:
need a space between Chinese and English

Reply:
I will fix it in another pr